### PR TITLE
chore: test against third party libraries

### PR DESCRIPTION
**Pull request by azjezz**
build failure is expected, and we can consider this parser to be ready to start using once it doesn't fail anymore :)

### CODE REVIEW

File: tests/third_party_tests.rs [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+#[test]
+fn third_party_php_standard_library() {
+    test_repository(
+        "php-standard-library",
+        "https://github.com/azjezz/psl.git",
+        "2.2.x",
+        &["src", "tests", "examples"],
+    );
+}
```
Comment (azjezz) [inline tests/third_party_tests.rs:17]: once tests pass for this, we can add more repositories.

**Comment by azjezz** [comment]
@ryangjchandler ready!

File: tests/third_party_tests.rs (hunk 1 above)
Comment (azjezz) [reply tests/third_party_tests.rs:17]: also a note: all third-party tests must start with `third_party` to ensure that they get filtered out

**Review by ryangjchandler — APPROVED** [review]

**Comment by ryangjchandler** [comment]
perfect!

**Pull request merged**